Lecture7-Zharova/src/com/company/HomeBle.java

```diff
@@ -0,0 +1,11 @@
+package com.company;
+
+public interface HomeBle {
```

Comment:
add a field to any interface as asked in task

Reply:
@ltana, fixed